src/Microsoft.SqlTools.Migration/SqlDataQueryController.cs

```diff
@@ -71,7 +71,10 @@ public class SqlDataQueryController : IDisposable
             this.messages = new List<KeyValuePair<string, DateTime>>();
             this.errors = new List<KeyValuePair<string, DateTime>>();
             perfDataCache = new SqlPerfDataPointsCache(this.outputFolder, _logger);
-            dataCollector = new DataPointsCollector(new string[] { connectionString }, _logger);
+            dataCollector = new DataPointsCollector(
```

Comment:
SqlDataQueryController is covered in test, and you are changing inside the constructor as hardcoded binary parameter. For now, it looks ok.